scripts/azkaban_job_whitelisting_blacklisting.py

```diff
@@ -0,0 +1,169 @@
+import mysql.connector
+import json
+import re
+import smtplib
+import logging
+import logging.handlers as handlers
+import sys
+
+
+if len (sys.argv) != 2 :
+    print "Usage: python azkaban_job_whitelisting_blacklisting.py config_file_name "
+    sys.exit (1)
+config_file_name = sys.argv[1]
+
+get_all_tuning_auto_apply_azkaban_rules_query="select rule_type, project_name, flow_name_expr, job_name_expr, job_type from tuning_auto_apply_azkaban_rules"
+get_all_jobs_for_a_project_query = "select jd.id, job_def_id, tjd.auto_apply, ta.job_type from job_definition jd inner join tuning_job_definition tjd on jd.id=tjd.job_definition_id inner join tuning_algorithm ta on tjd.tuning_algorithm_id=ta.id where job_def_id rlike %s"
+update_auto_apply_flag_query="""update tuning_job_definition set auto_apply=%s, updated_ts=current_timestamp where job_definition_id=%s"""
+db_config = None
+conn = None
+updated_job_definition_ids = {}
+updated_job_definition_ids[0]=[]
+updated_job_definition_ids[1]=[]
+BATCH_SIZE = 100
+
+logger=logging.getLogger() 
+logHandler = handlers.RotatingFileHandler('azkaban_job_whitelist.log', maxBytes=1073741824, backupCount=10)
+formatter = logging.Formatter('%(asctime)-12s [%(levelname)s] %(message)s')  
+logHandler.setFormatter(formatter)
+logger.setLevel(logging.INFO) 
+logger.addHandler(logHandler)
+
+logger.info("========================================================================")
+
+def get_mysql_connection():
+	db_config = json.load(open(config_file_name))
+	logger.info(db_config)
+	return mysql.connector.connect(**db_config)
+
+def get_project_name_expr(project_name):
+	return "project=" + project_name + "&"
+
+def get_all_jobs_for_a_project(project_name):
```

Comment:
Nit : Please add comment to describe functionality of the methods

Reply:
Done.